### ISSUE TITLE
TimeSpanUpDown ShowDays and ShowSeconds bug inside ListView.ItemTemplate

### ISSUE THREAD

**Issue by tqk2811** [xceedsoftware/wpftoolkit#1768]
Version: Extended.Wpf.Toolkit 4.6.0 ( latest )
------------------------------
```xaml
<!--top TimeSpanUpDown-->
 <Xceed:TimeSpanUpDown Value="{Binding Source={StaticResource TimeRange},Path=TimeRange}" ShowDays="False" ShowSeconds="False"/>

<!--List View-->
<ListView MinHeight="60" ScrollViewer.HorizontalScrollBarVisibility="Disabled"
          ItemsSource="{Binding Source={StaticResource TimeRange},Path=TimeList}" 
          d:ItemsSource="{d:SampleData ItemCount=10}">
    <ItemsControl.ItemsPanel>
        <ItemsPanelTemplate>
            <WrapPanel HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Margin="5" Orientation="Horizontal"/>
        </ItemsPanelTemplate>
    </ItemsControl.ItemsPanel>
    <ListView.ItemTemplate>
        <DataTemplate>
            <StackPanel Orientation="Horizontal">
                <CheckBox IsChecked="{Binding IsChecked}" VerticalAlignment="Center"/>
                <Xceed:TimeSpanUpDown Value="{Binding Time}" Maximum="23:59" Minimum="00:00"/>
            </StackPanel>
        </DataTemplate>
    </ListView.ItemTemplate>
</ListView>
```
![image](https://github.com/xceedsoftware/wpftoolkit/assets/13986296/17704312-8d61-4399-8aa0-856be4a6312a)
-----------------------------
When i change TimeSpanUpDown inside listview such as 
```xaml
<Xceed:TimeSpanUpDown Value="{Binding Time}" ShowDays="False"/>
<Xceed:TimeSpanUpDown Value="{Binding Time}" ShowSeconds="False"/>
<Xceed:TimeSpanUpDown Value="{Binding Time}" ShowDays="False" ShowHours="False"/>
```
No error msg in output visual studio  

![image](https://github.com/xceedsoftware/wpftoolkit/assets/13986296/dbb44451-d7c2-40d5-a3cb-a7e1f8d8875f)
-----------------------------

**Comment by XceedBoucherS** [comment]
Hello,

I've created a basic sample based on yours :

![image](https://github.com/xceedsoftware/wpftoolkit/assets/29377763/eb605cf4-5c83-405b-8ffb-e22e2f231527)


 public MainWindow()
 {
   InitializeComponent();

   this.DataContext = new List<TimeSpan>()
   {
     new TimeSpan(9, 11, 32),
     new TimeSpan(21, 5, 14),
   };
 }

It is working as expected with the latest v4.6 version of the Toolkit on NuGet:
![image](https://github.com/xceedsoftware/wpftoolkit/assets/29377763/613ed2dc-be90-4adc-a820-23c768014a0a)

I'm I missing something or it's in your sample ?
Thank you

**Comment by tqk2811** [comment]
![image](https://github.com/xceedsoftware/wpftoolkit/assets/13986296/cd1f64a1-112d-4e7b-96b2-2a26d560c4c2)

```xaml
<Window x:Class="WpfTestXceedTimeSpanUpDown.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfTestXceedTimeSpanUpDown"
        xmlns:Xceed="http://schemas.xceed.com/wpf/xaml/toolkit"
        mc:Ignorable="d"
        Title="MainWindow" Height="450" Width="800">
    <Window.DataContext>
        <local:MainWVM/>
    </Window.DataContext>
    <Grid>
        <StackPanel Orientation="Vertical" >
            <Xceed:TimeSpanUpDown Value="{Binding Path=TimeSpan}" ShowDays="False" ShowSeconds="False"/><!--(1)-->

            <ListView  MinHeight="60" ScrollViewer.HorizontalScrollBarVisibility="Disabled"
                    ItemsSource="{Binding TimeList}" 
                    d:ItemsSource="{d:SampleData ItemCount=10}">
                <ItemsControl.ItemsPanel>
                    <ItemsPanelTemplate>
                        <WrapPanel HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Margin="5" Orientation="Horizontal"/>
                    </ItemsPanelTemplate>
                </ItemsControl.ItemsPanel>
                <ListView.ItemTemplate>
                    <DataTemplate>
                        <StackPanel Orientation="Horizontal">
                            <CheckBox VerticalAlignment="Center"/>
                            <Xceed:TimeSpanUpDown Value="{Binding Time}" Maximum="23:59" Minimum="00:00" ShowDays="False"/><!--(2)-->
                        </StackPanel>
                    </DataTemplate>
                </ListView.ItemTemplate>
            </ListView>
        </StackPanel>
    </Grid>
</Window>
```

I found bropblem when (1) and (2)
https://github.com/tqk2811/WpfTestXceedTimeSpanUpDown

**Comment by XceedBoucherS** [comment]
Hi,

Thank you for the feedback.
I was able to reproduce the issue. It will be fixed in the next release.

If you want a quick fix, you can go in file
Xceed.Wpf.Toolkit/TimeSpanUpDown/Implementation/TimeSpanUpDown.cs
and replace all the occurences of
this.Value = something
to
this.SetCurrentValue( TimeSpanUpDown.ValueProperty, something);

Thank you

**Comment by tqk2811** [comment]
fixed in 4.6.1
